api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml

```diff
@@ -0,0 +1,1202 @@
+                      <ul>
+                        <li>updateCapacity</li>
+                      </ul>"
+                    example: 1
+                  adjustment:
+                    title: Adjustment
+                    description: "Value to which the action type will be adjusted. Required if using the next action types:
+                      <ul>
+                      <li>numeric</li>
+                      <li>percentageAdjustment</li>
+                      <ul>"
+                    type: string
+                    example: "2"
+                policyName:
+                  type: object
+                  example: "cpuScaleDown"
+                  threshold:
+                    example: 0.25
+                  operator:
+                    example: "lte"
+                  action:
+                    type: object
+                    properties:
+                      type:
+                        type: string
+                        example: adjustment
+                        enum:
+                          - adjustment
+                          - setMaxTarget
+                          - updateCapacity
+                          - percentageAdjustment
+      strategy:
+        type: "object"
+        title: Scaling Strategy
+        properties:
+          lowPriorityPercentage:
+            title: Low Priority Percentage
+            description: "Percentage of Low Priority instances to maintain.
+            </br><a style=color:red>Required if odCount isn't specified.</a>"
+            type: "integer"
+            default: 100
+          onDemandCount:
+            title: On Demand Count
+            type: integer
+            description: "Number of On-Demand instances to maintain.
+            </br><a style=color:red>Required if lowPriorityPercentage isn't specified.</a>"
+          drainingTimeout:
+            title: On Demand Count
+            type: integer
+            description: "Time (seconds) to allow the instance be drained from incoming TCP
+            connections and detached from MLB before terminating it during a scale down operation."
+            default: 120
+      compute:
+        title: Compute
+        description: Elastigroup's Compute Configurtion
+        type: "object"
+        required:
+          - vmSizes
+          - product
+          - launchSpecification
+        properties:
+          vmSizes:
+            title: VM Sizes
+            description: Sizes of On-Demand and Low-Priority VMs.
+            type: "object"
+            required:
+              - odSizes
+              - lowPrioritySizes
+            properties:
+              odSizes:
+                title: OD Sizes
+                type: string
+                description: "Array of string (s) with Available On-Demand sizes (pls chose items that correspond to your location or zone):
+                <ul>
+                    <li>standard_e64-32s_v4</li>
```

Comment:
I rather see a link to a page which specifies a variety of types instead of this very long list

Reply:
Asked several times to Yarden, but didn't got answer, so decided to put link on my own (https://docs.microsoft.com/en-us/azure/templates/ )

![Screenshot 2020-11-03 at 13 26 27](https://user-images.githubusercontent.com/64230439/97980495-8a47c480-1dd9-11eb-958c-8bd82839255e.png)

Reply:
odSizes - https://docs.microsoft.com/uk-ua/azure/virtual-machines/sizes
lowPrioritySizes - https://docs.microsoft.com/uk-ua/azure/virtual-machines/sizes 
namespace - https://docs.microsoft.com/en-us/azure/templates/

Reply:
Fixed. FYI I thing form user perspective it was better to have a list of acceptable values in OpenAPI because Azure require additional steps (filtering).